code/processes/housekeeping.q

```diff
@@ -74,13 +74,16 @@ kdbzip:{[FILE]
 \d .unix
 
 //-locates files with path, matching string and age
-find:{[path;match;age;agemin]
+find:{[path;match;age;agemin;checkfordirectory]
+	$[0=checkfordirectory;
+		[fileordir::"files";flag::"f"];
+		[fileordir::"directories";flag::"d"]];
```

Comment:
These shouldn't be global - can you make them local and pass them as params to the function that does the search.